fulp_modules/nightmare_edit/nightmare.dm

```diff
@@ -1,4 +1,7 @@
+/// Midround nightmare spawns are stupid.
 /datum/species/shadow/nightmare/spec_life(mob/living/carbon/human/H)
-	. = ..()
+	if(!H.mind.has_antag_datum(/datum/antagonist/nightmare))
```

Comment:
Make sure nightmare antagonist status is assigned **before** species is.

Reply:
Ive tested it and it works.